### PR TITLE
Milestone components

### PR DESCRIPTION
**Pull request by HADDADSOHAIB**
In this milestone, I added the skeleton components required to make a calculator. I added four component: Button, ButtonPanel, Display, and App.

### CODE REVIEW

File: src/components/Display.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    <div id="display">
+      The result is:
+      { result }
+    </div>
+  );
+};
+
+Display.propTypes = {
+  result: PropTypes.number,
+};
```
Comment (shloch) [inline src/components/Display.js]: `result` should be a string [ see [requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png) ]

File: src/components/Display.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Display.defaultProps = {
+  result: 0,
+};
+
```
Comment (shloch) [inline src/components/Display.js]: That means `defaultProps` value should be `"0"` [[requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png)]

**Review by shloch — CHANGES_REQUESTED** [review]
## Hello @HADDADSOHAIB , 

You did everything pretty correct. However for the sake of sticking with the guidelines, please you have some small debugging to do below prior to your last thumbs up

### Happy coding

**Comment by HADDADSOHAIB** [comment]
@shloch  thanks for the review, I fix it the issue :)

**Review by fatymahmed — APPROVED** [review]
## Hi @HADDADSOHAIB,
**Great job making the changes 🎉🎉**
## ✔️ Approved!

**Pull request merged**